src/main/resources/com/nec/cyclone/cpp/tests/nullable_scalar_vector_spec.hpp

```diff
@@ -26,11 +26,11 @@
 
 namespace cyclone::tests {
   TEST_SUITE("NullableScalarVec<T>") {
-    TEST_CASE_TEMPLATE("Equality checks work for T=", T, int32_t, int64_t, float, double) {
-      // Instantiate for int32_t
-      std::vector<T> raw { 586, 951, 106, 318, 538, 620, 553, 605, 822, 941 };
+    // Instantiate for each template case to be tested
+    template<class T> const std::vector<T> raw { 586, 951, 106, 318, 538, 620, 553, 605, 822, 941 };
```

Comment:
Seems a bit odd to say "class" when referring to integral types. I would have preferred "typename" here. But it appears that starting in C++17 the rules have been relaxed. It seems like typename and class are now fully interchangeable.

Reply:
That was a copy-paste error on my part.  Fixed